client/react/frontpage/components/Pagination.js

```diff
@@ -25,22 +25,18 @@ const Pagination = React.createClass({
   },
   goToNextPage() {
     if (this.nextIsValid()) {
-      this.props.setPageNumber(this.props.pageNumber + 1);
-      this.setState({
-        pageNumber: this.props.pageNumber + 1,
-      });
+      this.props.increasePageNumber();
     }
   },
   goToPrevPage() {
     if (this.prevIsValid()) {
-      this.props.setPageNumber(this.props.pageNumber - 1);
-      this.setState({
-        pageNumber: this.props.pageNumber - 1,
-      });
+      this.props.decreasePageNumber();
     }
   },
   nextIsValid() {
-    return this.props.pageNumber < this.props.maxPages;
+    let postsleft = this.props.postNumber - 12 * (this.props.pageNumber + 1);
```

Comment:
make `postsleft` camelcase to be consistent with the rest of the variables. camel case means to make it like `postsLeft`